substrate/frame/system/benchmarking/src/lib.rs

```diff
@@ -15,221 +15,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Benchmarks for Utility Pallet
+//! Frame System benchmarks.
 
 #![cfg_attr(not(feature = "std"), no_std)]
-#![cfg(feature = "runtime-benchmarks")]
```

Comment:
Same. This crate should only be activated when you are compiling runtime benchmarks.